pages/Documents/WebsiteMonitoring/MonitoringAPI/getting-started.md

```diff
@@ -9,8 +9,13 @@ order: 10
 permalink: monitoring-api-getting-started.html
 indicator: messaging
 ---
+### Step 1 - Retrieve your domain
 
-### Step 1 — Configure your Conversational Cloud account
+Use the [LivePerson Domain API](agent-domain-domain-api.html) to retrieve this information by providing the following service name:
```

Comment:
A list for one item seems unusual. How about incorporating this in the text, as with something like:

‘…by providing the service name “smt”.’

or

‘…by providing the following service name: “smt”.’

Reply:
I was trying to align it with the format that appears everywhere else in our documentation on the developer site such as here for the App Engagement API: https://developers.liveperson.com/app-engagement-api-overview.html

Reply:
We should change that everywhere, but it’s nothing we’d ask you to do. Fair enough.